pkg/api/v1/types.go

```diff
@@ -2060,3 +2060,83 @@ type RangeAllocation struct {
 	Range string `json:"range" description:"a range string that identifies the range represented by 'data'; required"`
 	Data  []byte `json:"data" description:"a bit array containing all allocated addresses in the previous segment"`
 }
+
+// SecurityContextConstraints governs the ability to make requests that affect the SecurityContext
+// that will be applied to a container.
+type SecurityContextConstraints struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty" description:"standard list metadata; see http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"`
+
```

Comment:
1. All these fields should be optional, so they need omitempty
2. Documentation is now specified via comments. Eliminate the description tags.